src/plugins/intel_gpu/src/kernel_selector/core/actual_kernels/roi_align/roi_align_kernel_ref.cpp

```diff
@@ -70,21 +72,36 @@ bool ROIAlignKernelRef::Validate(const Params& p, const optional_params& o) cons
     if (params.inputs.size() != 3)
         return false;
 
-    if (params.outputs[0].Dimentions() > 4 || params.inputs[0].Dimentions() > 4 || params.inputs[1].Dimentions() > 2)
```

Comment:
why have you removed it if by definition output is 4-dimension? Or blocked layouts return another value?

Reply:
Yes, it was removed due to blocked formats